app/controllers/tag_wranglers_controller.rb

```diff
@@ -1,6 +1,8 @@
 class TagWranglersController < ApplicationController
+  include ExportsHelper
+
   before_action :check_user_status
-	before_action :check_permission_to_wrangle
+  before_action :check_permission_to_wrangle, except: [:report_csv]
```

Comment:
What would be the use case in which a superadmin or tag wangler is allowed to access `report_csv` without validating at the same time `check_permission_to_wrangle`?

Reply:
I don't think it would cause problems to do both, but it seems like unnecessary duplication to have two authorization checks

Reply:
Yeah, I think we can keep the `except: [:report_csv]` here for now, at least until we have a better idea what `check_permission_to_wrangle` will look like when we lock admin wrangling down more.